default.conf

```diff
@@ -2,6 +2,10 @@ server {
     listen 80;
     server_name  _;
 
+    if ($host ~ "oauth.deriv.app") {
```

Comment:
This is not recommended by NGINX https://www.nginx.com/blog/creating-nginx-rewrite-rules/.

Reply:
The above method can also have unintended consequences and it is hard to debug. I think a simple location directive for / to redirect to deriv.app should be good.

Reply:
that may cause too much redirection loop in nginx which was faced recently for deriv.com. so i am creating another server block for oauth.deriv.app and rewriting url to `/oauth2/`